site/docs/guide/deployment-types.md

```diff
@@ -30,15 +30,15 @@ _Imagine you're getting yourself a scoop of ice cream in your trusted ice cream
 You walk up to the employee and ask for your favorite type of ice cream.
 Unfortunately, he lets you know that it is out of stock._
 
-_The next day, you're craving that delicious ice cream again, so you go back to the same place and ask for the same ice cream.
+_The next day, you're craving that delicious ice cream again, so you return to the same place and ask for the same ice cream.
 Good news!
-They restocked over night so you can enjoy your salted caramel ice cream today!
+They are restocked overnight, so you can enjoy your salted caramel ice cream today!
```

Comment:
This does not make sense. The employees were not restocked, it was them who did the restocking.